app/helpers/react_components/mentoring/discussion.rb

```diff
@@ -9,8 +9,18 @@ def to_s
           {
             discussion_id: discussion.uuid,
             student: {
+              name: student.name,
+              handle: student.handle,
+              bio: student.bio,
+              languages_spoken: student.languages_spoken,
               avatar_url: student.avatar_url,
-              handle: student.handle
+              reputation: student.reputation,
+              is_favorite: student.favorited_by?(mentor),
+              num_previous_sessions: mentor.num_previous_mentor_sessions_with(student),
+              links: {
+                # TODO
+                favorite: "stub_link"
```

Comment:
@iHiD I'm assuming that the link to add a favorite and delete a favorite is the same and they only differ with the HTTP action. There is no implementation yet so I did it this way. We can change it in the future if there are two different links.

Reply:
Yeah. I'll add these end points today for you.